File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int counts[] = {0, 0, 0};

        for (int x : nums) {
            counts[x]++;
        }

        int i = 0;
        for(int n = 0; n < 3; n++) {
            for(int j = 0; j < counts[n]; j++) {
                nums[i] = n;
                i++;
            }
        }
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        counter = 0
        res = 0
        for i in range(len(nums)):
            if nums[i] == 0:
                res = max(res, counter)
                counter = 0
                continue
            counter += 1
        
        res = max(res, counter)
        
        return res